src/structures/BaseGuildEmoji.js

```diff
@@ -51,6 +51,8 @@ class BaseGuildEmoji extends Emoji {
      */
     if (typeof data.available !== 'undefined') this.available = data.available;
 
+    if (typeof data.user !== 'undefined') this.author = this.client.users.add(data.user);
```

Comment:
I think this should rather be in `GuildEmoji#_patch`, as preview emojis will never hold this information.